src/material/form-field/form-field.scss

```diff
@@ -63,10 +63,14 @@
   width: 100%;
 }
 
-// The MDC text-field should stretch to the width of the host `<mat-form-field>` element.
-// This allows developers to control the width without needing custom CSS overrides.
 .mat-mdc-text-field-wrapper {
```

Comment:
This element wraps the entire form field. Maybe we should only apply `pointer-events: auto` to the prefix and suffix containers?

Reply:
They're kind of contrived examples, but I can think of two cases:
1. Somebody could implement a custom form control which is something like `<div tabindex="-1"></div>` which would be focusable even if it's technically disabled. The `pointer-events: none` would prevent that.
2. We have the `onContainerClick` on the `MatFormFieldControl` which would be blocked by `pointer-events`.

Reply:
Yeah, I think this came from MDC from the beginning, but it was useful because of the custom form field use-case Kristiyan brought up. I don't recall if the old form-field did the same- but I had a feeling it had `pointer-events: none` too

Reply:
The container click is a good point, I forgot about that. Ok, I'll switch it to just allow clicking on the prefix/suffix